README.md

```diff
@@ -177,6 +179,8 @@ FETCH_LABELS=
 ```
 # Update document links, so they point to their synced document counterpart in destination instead of source. 
 UPDATE_DOCUMENT_LINKS=
+# Generate and attach an external url field to the articles that will point to the article in the source system. Default false
```

Comment:
I think we should follow the pattern that is used at Fetch articles' description:
`Values: 'true' or 'false'. Default 'false'. `